operations/mimir-mixin/dashboards/overview-resources.libsonnet

```diff
@@ -0,0 +1,95 @@
+local utils = import 'mixin-utils/utils.libsonnet';
+local filename = 'mimir-overview-resources.json';
+
+(import 'dashboard-utils.libsonnet') +
+(import 'dashboard-queries.libsonnet') {
+  [filename]:
+    ($.dashboard('Overview resources') + { uid: std.md5(filename) })
+    .addClusterSelectorTemplates()
```

Comment:
There was something about the resource panels that didn't allow for multiple selectors for the cluster. So other resource dashboards have `.addClusterSelectorTemplates(false)`. I haven't looked in why that is

Reply:
Nicely spot! We don't want to allow to select multiple (or all) namespaces otherwise these panels may explode (since they all show per-pod metrics). I've fixed it in https://github.com/grafana/mimir/pull/3481/commits/6e52309f31fe07284864ef80adc185ff8dff07ac so that it's consistent with the other resources dashboard.